plugins/versionpress/tests/Automation/WpAutomation.php

```diff
@@ -557,12 +557,17 @@ private function prepareFiles()
     private function ensureCleanInstallationIsAvailable()
     {
 
-        if (!is_dir($this->getCleanInstallationPath())) {
-            $downloadPath = $this->getCleanInstallationPath();
-            FileSystem::mkdir($downloadPath);
+        $cleanInstallationPath = $this->getCleanInstallationPath();
+
+        if (!is_dir($cleanInstallationPath) || !is_file($cleanInstallationPath . '/wp-settings.php')) {
+            if (!is_dir($cleanInstallationPath)) {
```

Comment:
```php
if (!is_dir($cleanInstallationPath) || !is_file($cleanInstallationPath . '/wp-settings.php')) {
    FileSystem::remove($cleanInstallationPath);
    FileSystem::mkdir($cleanInstallationPath);

    $wpVersion = $this->siteConfig->wpVersion;
    ...
```

What about this? It looks more straightforward to me. Maybe we could also extract `is_file($cleanInstallationPath . '/wp-settings.php')` into method like `isCorrectlyDownloaded` or something like that with a comment about this good-enough solution.